lib/grape/validations/validators/length_validator.rb

```diff
@@ -18,6 +18,8 @@ def initialize(attrs, options, required, scope, **opts)
         def validate_param!(attr_name, params)
           param = params[attr_name]
 
+          return if param.nil?
```

Comment:
shouldn't this only return if the `required` field is false. We need to store that in an instance member variable now as well.

Reply:
Sorry, I don't understand that return here will lead to problems such as instance variables. If possible, could you please explain it more clearly?

Reply:
sorry for not being clear 😅 . I was wondering if we should be skipping the validation for required parameters or not ?

After thinking about it a `nil` required parameter would fail validation anyways so this should be fine ✅

Reply:
Thank～